apps/api/src/app/models/mentors/mentor.service.ts

```diff
@@ -2,10 +2,14 @@ import { Injectable } from '@nestjs/common';
 import { InjectModel } from '@nestjs/mongoose';
 import { Model } from 'mongoose';
 import { ViewModels } from '../../helpers/constants';
-import { MentorViewModel } from './mentor.interface';
+import { MentorViewModel, CreateMentor } from './mentor.interface';
+import { MentorCreated } from '../../domains/mentors/events/mentor-created.event';
 
 @Injectable()
 export class MentorModelService {
   @InjectModel(ViewModels.MENTORS) public model!: Model<MentorViewModel>;
 
+  public async createMentor(data: MentorCreated): Promise<void> {
```

Comment:
You can change `data` to` {  id, mentor }`, so you could use id and mentors without data in front.